python/taichi/lang/impl.py

```diff
@@ -126,23 +126,34 @@ def wrap_scalar(x):
 @taichi_scope
 def subscript(value, *_indices, skip_reordered=False):
     if isinstance(value, np.ndarray):
-        return value.__getitem__(*_indices)
+        return value.__getitem__(_indices)
```

Comment:
Why this change?

Reply:
`ndarray.__getitem__` only supports an index or a tuple of indices, the previous one has a bug then deal with multiple indexes